lib/order.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'text'
+require_relative 'menu'
+
+class Order
+  include Menu
+
+
+  attr_accessor :contents, :add_item
+
+  def initialize
+    @contents = []
+  end
+
+  def add_item(dish, quantity)
+    if display_menu.has_key?(dish)
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`.